packages/portal-proto/src/components/Table/VerticalTable.tsx

```diff
@@ -169,6 +177,25 @@ function VerticalTable<TData>({
     handleChange({ newSearch: "" });
   };
 
+  const tooltipContainer = search?.tooltip
```

Comment:
Nitpick: Component name should be PascalCased. 
```suggestion
  const TooltipContainer = search?.tooltip
```